starboard/nplb/compiler_compliance/cpp17_support.cc

```diff
@@ -17,18 +17,31 @@
 #include "starboard/configuration.h"
 #include "starboard/types.h"
 
+namespace foo::bar::baz {
```

Comment:
Perhaps make this a `starboard::foo::bar::baz` so it matches the module name and doesn't "leak" into symbols